### DISCUSSION TITLE
Wix 5 - File in use despite not being harvested

### DISCUSSION THREAD

**Discussion by ericnev** [wixtoolset/issues#8699] [Questions]
I have a Wix3 installation that I've recently migrated to Wix5, and am taking advantage of the new Files element to simplify (formerly Heat-based) harvesting.

My structure looks basically like this:

```
/File1
/File2
/File3
..
/Dir
/Dir/File1
/Dir/File2
..
/LauncherDir
/LauncherDir/File1
/LauncherDir/Launcher.exe
..
```

The applicable portion of WXS looks essentially like:

```
<?define MainOutputDir = $(var.SolutionDir)/MyProject/bin/$(var.Configuration)/ ?>
<ComponentGroup Id="Files" Directory="INSTALLFOLDER">
			<File Id="Main.App.EXE" Source="$(var.MainOutputDir)\Main.App.exe" />
			<Files Directory="INSTALLFOLDER" Include="$(var.MainOutputDir)/**" >
				<Exclude Files="$(var.MainOutputDir)\Main.App.exe" />
				<Exclude Files="$(var.MainOutputDir)\**.tmp" />
				<Exclude Files="$(var.MainOutputDir)\**.pdb" />
				<Exclude Files="$(var.MainOutputDir)\**.xml" />
				<Exclude Files="$(var.MainOutputDir)\LauncherDir\**" />
			</Files>
		</ComponentGroup>
```

The installer is launched by **/LauncherDir/Launcher.exe** and that's **explicitly** excluded from harvesting, but the installer still wants to kill it, seemingly because it lives in the root directory that the installer is updating? Is this behavior intentional?

**Comment by barnson** [comment]
Did the file end up in the .msi?

**Reply by ericnev** [reply]
Kind of. 

I'm rolling out a copy of said file, but in another directory (**LauncherDir.Update/Launcher.exe**), which was expected. My surprise was that MSI/Windows wanted to kill my existing process, despite the exe being written to a directory that shouldn't have been being touched, if that makes sense.
